### PR TITLE
fixed: do not use integer abs

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this no downstreams please

File: test_util/test_esmry_lod.cpp [hunk 1]
```diff
@@ -110,7 +110,7 @@ int main(int argc, char **argv) {
 
             for (size_t m = 0; m < v1.size(); m++) {
 
-                float absDiff = abs( v1[m]- v2[m]);
+                float absDiff = std::abs( v1[m]- v2[m]);
```
Comment (bska) [inline test_util/test_esmry_lod.cpp:113]: Probably need to
```C++
#include <cmath>
```
too...

Comment (akva2) [reply test_util/test_esmry_lod.cpp:113]: Builds fine so must already be included through some other header. But i can add.

**Comment by akva2** [comment]
jenkins build this no downstreams please

File: test_util/test_esmry_lod.cpp (hunk 1 above)
Comment (bska) [reply test_util/test_esmry_lod.cpp:114]: > Builds fine so must already be included through some other header.

Probably.  We should not rely on that though.

> But i can add.

Thanks.  I'll merge into master.

**Pull request merged**